gloo/test/multiproc_test.h

```diff
@@ -64,18 +65,27 @@ class MultiProcWorker {
         new ::gloo::rendezvous::FileStore(storePath));
     semaphore_ = sem_open(semaphoreName.c_str(), 0);
     GLOO_ENFORCE_NE(semaphore_, (sem_t*)nullptr, strerror(errno));
+    cerf_file_path = std::string(std::tmpnam(nullptr)); // revert!
+    pkey_file_path  = std::string(std::tmpnam(nullptr)); // revert!
+    create_pkey_and_cert(pkey_file_path, cerf_file_path); // revert!
   }
 
   ~MultiProcWorker() {
     sem_close(semaphore_);
+    std::remove(pkey_file_path.c_str()); // revert!
```

Comment:
why its default dtor does not work?

Reply:
sorry, I don't understand what you mean, it's deletion of `pkey_file_path` file